tasks/sep/016-fp-audit-fix/NestedSignFromJson.s.sol

```diff
@@ -0,0 +1,166 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+import {NestedSignFromJson as OriginalNestedSignFromJson} from "script/NestedSignFromJson.s.sol";
+import {OptimismPortal2, IDisputeGame} from "@eth-optimism-bedrock/src/L1/OptimismPortal2.sol";
+import {Types} from "@eth-optimism-bedrock/scripts/Types.sol";
+import {Vm, VmSafe} from "forge-std/Vm.sol";
+import {console2 as console} from "forge-std/console2.sol";
+import {stdToml} from "forge-std/StdToml.sol";
+import {LibString} from "solady/utils/LibString.sol";
+import {GnosisSafe} from "safe-contracts/GnosisSafe.sol";
+import "@eth-optimism-bedrock/src/dispute/lib/Types.sol";
+import {ISemver} from "@eth-optimism-bedrock/src/universal/ISemver.sol";
+import {FaultDisputeGame} from "@eth-optimism-bedrock/src/dispute/FaultDisputeGame.sol";
+import {PermissionedDisputeGame} from "@eth-optimism-bedrock/src/dispute/PermissionedDisputeGame.sol";
+import {DisputeGameFactory} from "@eth-optimism-bedrock/src/dispute/DisputeGameFactory.sol";
+
+interface IASR {
+    function superchainConfig() external view returns (address superchainConfig_);
+}
+
+contract NesteSignFromJson is OriginalNestedSignFromJson {
```

Comment:
yikes. I don't even know how this worked before.
cc: @mds1 I'd expect post-check scripts with compilation errors should prevent the simulation from running.

Reply:
In the justfile we reference the script to run by file path + function signature, and the file path is correct here, so this typo is essentially a no-op.

If the typo was in the filename though, we would have ran the version without post-checks. However, that version deliberately has revert statements so it would have failed:

https://github.com/ethereum-optimism/superchain-ops/blob/988907dca62e26c4b70c66a1aa6704d3b2732f78/script/NestedSignFromJson.s.sol#L40-L48